src/main/java/seedu/address/model/ReadOnlyCalendar.java

```diff
@@ -0,0 +1,17 @@
+package seedu.address.model;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.event.Event;
+
+/**
+ * Unmodifiable view of an address book
+ */
+public interface ReadOnlyCalendar {
+
+    /**
+     * Returns an unmodifiable view of the persons list.
+     * This list will not contain any duplicate persons.
+     */
```

Comment:
Update Javadocs to specify it is about events not persons

Reply:
Done